### ISSUE TITLE
Add hotkeys support

### ISSUE THREAD

**Issue by Williangalvani** [mavlink/qgroundcontrol#7490]
Even though QGC is multi-platform, I don't see any harm in allowing navigation and some level of operation operation by using hotkeys when a keyboard is present.

It could be helpful for things like switching between Views, opening the widgets, operating the cameras, changing parameters, placing waypoints in Plan view(#4275), and likely even more.  

Has this topic been discussed before? I couldn't find anything.

**Comment by DonLakeFlyer** [comment]
YEs it has been discussed and I always run into Qt's keyboard support being next to impossible to make work correctly. Just getting Esc/Enter to work on dialogs has been a nightmare.
